src/event-mixin.spec.ts

```diff
@@ -76,3 +76,67 @@ describe('a child class', () => {
     expect(eventThreeArgs).toHaveLength(1);
   });
 });
+
+// New test cases for the path functionality
+describe('DeepNestedEvents with path', () => {
+  class DeepNestedEvents {
+    eventContainer = {
+      deeper: {
+        eventOne: new TypedEvent<(value: number) => void>(),
+        eventTwo: new TypedEvent<(value: boolean) => void>(),
+      },
+    };
+
+    fireEventOne() {
+      this.eventContainer.deeper.eventOne.emit(42);
+    }
+
+    fireEventTwo() {
+      this.eventContainer.deeper.eventTwo.emit(true);
+    }
+  }
+
+  const NestedEventsClass = AddEvents<
+    typeof DeepNestedEvents,
+    {
+      eventOne: TypedEvent<(value: number) => void>;
+      eventTwo: TypedEvent<(value: boolean) => void>;
+    }
```

Comment:
nit: for consistency, can you extract these out into an interface like the test above does? with them matching it makes it easier to see that what's different here is the path argument.

Reply:
Sure thing. Done.